posthog/cdp/templates/meta_ads/template_meta_ads.py

```diff
@@ -9,22 +9,31 @@
     icon_url="/static/services/meta-ads.png",
     category=["Advertisement"],
     hog="""
+let body := {
+    'data': [
+        {
+            'event_name': inputs.eventName,
+            'event_time': inputs.eventTime,
+            'action_source': inputs.actionSource,
+            'user_data': {}
+        }
+    ],
+    'access_token': inputs.accessToken
+}
+
+for (let key, value in inputs.userData) {
+    // e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855 is an empty string hashed
+    if (not empty(value) and value != 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855') {
```

Comment:
Generally I really don't like this fix. Having a big hardcoded string in every function feels very wrong. I get why it is here but feels like it would be way nicer if the `sha256Hex()` function would return null or something if given an empty value. Appreciate thats outside of the scope of this so we can move forward but would be interesting to see if @mariusandra has alternative ideas.